google/resource_compute_instance_group_manager.go

```diff
@@ -422,6 +442,18 @@ func resourceComputeInstanceGroupManagerRead(d *schema.ResourceData, meta interf
 	d.Set("update_strategy", update_strategy.(string))
 	d.Set("auto_healing_policies", flattenAutoHealingPolicies(manager.AutoHealingPolicies))
 
+	if d.Get("wait_for_instances").(bool) {
+		conf := resource.StateChangeConf{
+			Pending: []string{"creating", "error"},
+			Target:  []string{"created"},
+			Refresh: waitForInstancesRefreshFunc(getManager, d, meta),
+			Timeout: d.Timeout(schema.TimeoutCreate),
+		}
+		_, err := conf.WaitForState()
+		// If err is nil, success.
```

Comment:
If someone later adds some code below this, it'll get missed - might be good to add a comment saying not to do that?  Not sure.

Reply:
Just updated it to return the error only if there is one, that seemed like a much more standard way to go about this.